CIPs/0012.md

```diff
@@ -0,0 +1,39 @@
+# CIP 0012: More granular validator signer keys
+
+- Date: 2020-05-20
+- Author: @kobigurk
+- Status: WIP
```

Comment:
hi @kobigurk  thank you for submitting this CIP, I would like you to update it to follow the newly activated CIP process which has it's newly updated template here: https://github.com/celo-org/celo-proposals/blob/master/CIPs/cip-template.md

Also, if the CIP is in Draft status, I'd be able to merge the proposal into the repo.